### PR TITLE
Basler urdf parameter

### PR DESCRIPTION
**Pull request by Flova**
This pull request adds a parameter (the default is true) to select the Basler camera in the teamplayer and the motion_standalone launch file. By setting this parameter the correct urdf for the Basler camera is loaded.

### CODE REVIEW

File: bitbots_bringup/launch/motion_standalone.launch [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <launch>
     <arg name="sim" default="false"/>
+    <arg name="basler" default="true"/>
```
Comment (NFiedler) [inline bitbots_bringup/launch/motion_standalone.launch]: you could add parameter documentation here.

File: bitbots_bringup/launch/load_robot_description.launch [hunk 2]
```diff
@@ -8,6 +8,8 @@
     <arg name="davros_length" default="0" />
     <!-- fake walk controller for gazebo -->
     <arg name="use_fake_walk" default="False" />
+    <!-- defines camera type -->
+    <arg name="basler" default="true" />
```
[the remark this answers is not in the document]
Comment (Flova) [reply bitbots_bringup/launch/load_robot_description.launch]: What kind of documentation do you mean?
Should I add line 11 to the other files where the parameter is used?

File: bitbots_bringup/launch/motion_standalone.launch (hunk 1 above)
Comment (NFiedler) [reply bitbots_bringup/launch/motion_standalone.launch]: Actually, there is an actual element for the XML: https://wiki.ros.org/roslaunch/XML/arg

File: bitbots_bringup/launch/load_robot_description.launch (hunk 2 above)
[the remark this answers is not in the document]
Comment (NFiedler) [reply bitbots_bringup/launch/load_robot_description.launch]: There is an XML element for this. https://wiki.ros.org/roslaunch/XML/arg

File: bitbots_bringup/config/camera_settings.yaml [hunk 3]
```diff
@@ -109,11 +109,11 @@ gamma: 1.0
 #  with the MTU size the network card supports. A value greater 3000
 #  should be good (1500 for RaspberryPI)
 # gige:
-gige/mtu_size: 1500
+gige/mtu_size: 3000
```
Comment (timonegk) [inline bitbots_bringup/config/camera_settings.yaml]: The changes in this file do not belong to this PR (and have already been updated on master today), please remove them.

File: bitbots_bringup/launch/load_robot_description.launch [hunk 4]
```diff
@@ -20,8 +22,13 @@
 
 
     <group if="$(arg wolfgang)">
-        <param name="robot_description" command="$(find xacro)/xacro $(find wolfgang_description)/urdf/wolfgang.urdf.xacro use_fake_walk:=$(arg use_fake_walk) sim_ns:=$(arg sim_ns) --inorder" />                
-        <param name="urdf" value="$(find wolfgang_description)/urdf/wolfgang.urdf.xacro" />
+        <param name="robot_description" command="$(find xacro)/xacro $(find wolfgang_description)/urdf/wolfgang_basler.urdf.xacro use_fake_walk:=$(arg use_fake_walk) sim_ns:=$(arg sim_ns) --inorder" />
```
Comment (timonegk) [inline bitbots_bringup/launch/load_robot_description.launch]: Is it correct that the basler urdf is always used for the robot description?

**Comment by Flova** [comment]
> The Basler argument should additionally be added to the highlevel launch file where it is passed to the vision. 

But the vision runs on the jetson and is not started via the highlevel launch file. Except we would use the `<machine>` Tag.

File: bitbots_bringup/launch/load_robot_description.launch (hunk 4 above)
Comment (Flova) [reply bitbots_bringup/launch/load_robot_description.launch]: Oh, I oversaw this parameter.

**Comment by timonegk** [comment]
> But the vision runs on the jetson and is not started via the highlevel launch file. 
The vision is started if `vision:=true` ist set. The basler argument could be added there (however I concede that I did not remember that the vision is usually not started on the nuc)
